.rubocop.yml

```diff
@@ -4,3 +4,13 @@ LineLength:
    Max: 99
 Style/FrozenStringLiteralComment:
    Enabled: false
+Style/SymbolArray:
+  Enabled: false
+Rails/Blank:
+  Enabled: false
+Layout/TrailingBlankLines:
+  Enabled: false
+Style/SafeNavigation:
+  Enabled: false
```

Comment:
Also this one, this issue is really useful.
Info: https://www.rubydoc.info/gems/rubocop/0.43.0/RuboCop/Cop/Style/SafeNavigation
The safe navigator `&` works something like this:
```
user = User.find(1)
other_user = nil

user.name  # This will work, and will return the name of the user
other_user.name # This will blow up, because other_user is nil.
other_user&.name # This will work, and will return nil. The & check if the value of the variable, if it is nil, will return nil, if not, will return the value of the name
```